cos_alerter/alerter.py

```diff
@@ -186,6 +199,12 @@ def notify(self):
         notify_thread.start()
 
 
+def now_datetime():
+    """Return the datetime for now using the monotonic clock."""
```

Comment:
is "the current datetime" better?

also: `def now()` would be in line with datetime's API.

Reply:
Done.